pkgs/development/compilers/wren/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, python3
+, callPackage
+}:
+let
+  cli =
+    callPackage ./cli.nix { };
+in
+stdenv.mkDerivation rec {
+  pname = "wren";
+  version = "0.4.0";
+
+  src = fetchFromGitHub {
+    owner = "wren-lang";
+    repo = pname;
```

Comment:
`repo = pname;` is nice for DRY but creates a binding that goes too far.

See further information about this here: https://github.com/nix-community/nixpkgs-lint/issues/21